plutus-tx/test/Spec.hs

```diff
@@ -0,0 +1,43 @@
+module Main(main) where
+
+import           Codec.Serialise        (deserialiseOrFail, serialise)
+import           Hedgehog               (MonadGen, Property, annotateShow, assert, forAll, property)
+import qualified Hedgehog.Gen           as Gen
+import qualified Hedgehog.Range         as Range
+import           Language.PlutusTx.Data (Data (..))
+import           Test.Tasty
+import           Test.Tasty.Hedgehog    (testProperty)
+
+main :: IO ()
+main = defaultMain tests
+
+tests :: TestTree
+tests = testGroup "plutus-tx" [
+    serdeTests
```

Comment:
Serde?

Reply:
Sounds like a French swear word but it is commonly used to mean "serialisation & deserialisation" (in Scala land and apparently in Rust as well)